lib/logger/logger.ts

```diff
@@ -34,6 +36,11 @@ class Logger {
     this.logWriter = output;
   }
 
+  setRoutingConfig(filename, cb) {
+    this.logRouter = new router.Router();
+    this.logRouter.loadConfig(filename, cb);
```

Comment:
Another possible approach: just read the routes from the file directly:

```js
const routes = yaml.safeLoad(fs.readFileSync(filename)).routes
const logRouter = new router.Router(routes);
```

where `new router.Router` does validation on routes